libs/ledgerjs/script/crypto-assets-importer/importers/internet_computer/index.ts

```diff
@@ -0,0 +1,36 @@
+import { log } from "console";
+import fs from "fs";
+import path from "path";
+
+export const importInternetComputerTokens = async (outputDir: string) => {
+  log("importing internet computer tokens...");
+  const filePath = path.join(outputDir, "internet_computer");
+  fs.writeFileSync(
+    `${filePath}.ts`,
+    `export type ICPToken = [
+    string, // assetCode
+    string, // assetIssuer
+    // Note: asset type is only used in Receive asset message and always should be "InternetComputer"
+    string, // asset type
+    string, // name
+    number, // precision
+    boolean // enableCountervalues
+];
+
+const tokens: ICPToken[] = [
+  [
+    "USDC",
+    "GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN",
+    "InternetComputer",
+    "USDC",
+    7,
+    true,
+  ],
+];
```

Comment:
This works, but maybe better to respect the same file pattern as others?

Reply:
Not sure to understand what file pattern you're mentioning ?

Reply:
Sorry I meant putting the part I highlighted in its own .json file, while keeping this in a .ts file